### ISSUE TITLE
send from alias address -> NPE

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-android#184]
To reproduce:
1) compose secure email
2) attach a file
3) switch to plain-text
4) send
5) NPE

```
[STACK_TRACE]
java.lang.NullPointerException: Attempt to invoke virtual method 'byte[] java.lang.String.getBytes()' on a null object reference
at com.flowcrypt.email.api.email.sync.tasks.SendMessageSyncTask.runSMTPAction(SendMessageSyncTask.java:127)
at com.flowcrypt.email.api.email.sync.EmailSyncManager$BaseSyncRunnable.runSyncTask(EmailSyncManager.java:499)
at com.flowcrypt.email.api.email.sync.EmailSyncManager$ActiveSyncTaskRunnable.run(EmailSyncManager.java:559)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:428)
at java.util.concurrent.FutureTask.run(FutureTask.java:237)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
at java.lang.Thread.run(Thread.java:761)
```

```
java.lang.Exception: EmailSyncService.onError
at com.flowcrypt.email.service.EmailSyncService.onError(EmailSyncService.java:434)
at com.flowcrypt.email.api.email.sync.tasks.BaseSyncTask.handleException(BaseSyncTask.java:80)
at com.flowcrypt.email.api.email.sync.tasks.SendMessageSyncTask.handleException(SendMessageSyncTask.java:73)
at com.flowcrypt.email.api.email.sync.EmailSyncManager$BaseSyncRunnable.runSyncTask(EmailSyncManager.java:513)
at com.flowcrypt.email.api.email.sync.EmailSyncManager$ActiveSyncTaskRunnable.run(EmailSyncManager.java:559)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:428)
at java.util.concurrent.FutureTask.run(FutureTask.java:237)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
at java.lang.Thread.run(Thread.java:761)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'byte[] java.lang.String.getBytes()' on a null object reference
at com.flowcrypt.email.api.email.sync.tasks.SendMessageSyncTask.runSMTPAction(SendMessageSyncTask.java:127)
at com.flowcrypt.email.api.email.sync.EmailSyncManager$BaseSyncRunnable.runSyncTask(EmailSyncManager.java:499)
... 6 more
java.lang.NullPointerException: Attempt to invoke virtual method 'byte[] java.lang.String.getBytes()' on a null object reference
at com.flowcrypt.email.api.email.sync.tasks.SendMessageSyncTask.runSMTPAction(SendMessageSyncTask.java:127)
at com.flowcrypt.email.api.email.sync.EmailSyncManager$BaseSyncRunnable.runSyncTask(EmailSyncManager.java:499)
at com.flowcrypt.email.api.email.sync.EmailSyncManager$ActiveSyncTaskRunnable.run(EmailSyncManager.java:559)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:428)
at java.util.concurrent.FutureTask.run(FutureTask.java:237)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
at java.lang.Thread.run(Thread.java:761)
```

**Comment by tomholub** [comment]
Correction - it happens anytime I try to send email from a Gmail alias (human@flowcrypt.com)

**Comment by tomholub** [comment]
@DenBond7 this should be fixed in 0.4.2

**Comment by DenBond7** [comment]
The similar issue #153. But now I have more info.
